game/tests.py

```
@@ -1474,6 +1489,205 @@ def test_promotion_notation_invalid_piece_defaults_to_queen(self):
+
+    @override_settings(
+        DEBUG=True,
+        EMAIL_HOST_USER='',
+        EMAIL_HOST_PASSWORD='',
+    )
+    def test_inactive_username_is_preserved(self):
+        """Inactive accounts must never be deleted during registration."""
+        inactive = User.objects.create_user(
+            username='newchessplayer',
+            email='old@example.com',
+            password='OldPassword456!',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)